build.gradle.kts

```diff
@@ -280,19 +278,22 @@ allprojects {
                     url = uri(if (isSnapshot) "https://aws.oss.sonatype.org/content/repositories/snapshots/"
                         else "https://aws.oss.sonatype.org/service/local/staging/deploy/maven2")
                     credentials {
-                        username = "${findProperty("aws.sonatype.username")}"
-                        password = "${findProperty("aws.sonatype.password")}"
+                        username = "${findProperty("aws.sonatype.username") ?: System.getenv("SONATYPE_USERNAME")}"
+                        password = "${findProperty("aws.sonatype.password") ?: System.getenv("SONATYPE_PASSWORD")}"
                     }
                 }
             }
         }
 
         tasks.withType<Sign>().configureEach {
-            onlyIf { !isSnapshot }
+            onlyIf { System.getenv("CI") == "true" }
```

Comment:
So this will make releasing from a local machine not really possible right? That's fine with me, I just want to understand this correctly

Reply:
If we need to publish from local machine, we can set up the GPG env vars and CI = true locally too in the off event that we need to